### ISSUE TITLE
Installer

### ISSUE THREAD

**Issue by eltoncezar** [eltoncezar/Spofy#1]
I want to create an installer (and and uninstaller of course), to make life easier.

**Comment by blinkertoon** [comment]
Would love someone to do this - but meantime, any instructions on how to get this up and running??

**Comment by eltoncezar** [comment]
You can download the latest release from here: [https://github.com/eltoncezar/Spofy/releases/latest](https://github.com/eltoncezar/Spofy/releases/latest). Just download the .EXE and run it.

Also, I'm having second thoughts on the installer. Is it really needed?

**Comment by beppe9000** [comment]
> Also, I'm having second thoughts on the installer. Is it really needed?

I don't think so. An installer program makes sense when you have multiple support files or a complex set of requirements. Having an installation program for a standalone executable file is overkill.

I think that a better solution than a classical installer could be to include an "integration" submenu with the following items, that basically are what an installer usually does, or allows to do:
 - add link to start menu
 - make a copy of the spofy.exe in %program files%\spofy\spofy.exe and switch to that(close current instance and relaunch).
 - auto start on login
 - reset settings
 - delete all traces of spofy from the system (removes any spofy files from system paths (such as %programfiles% or %appdata%))
